aries-site/src/pages/components/accordion.js

```diff
@@ -18,23 +19,72 @@ const Accordion = () => (
-          entire header can be clicked to expand or collapse content.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
         <Example
-          details={[
-            `When seeking to provide maximum content in limited, vertical 
-            space, an accordion is a good alternative.`,
-          ]}
           docs="https://v2.grommet.io/accordion#props"
           code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/AccordionExample.js"
+          figma="https://www.figma.com/file/UvTBUQdhk07wV79nijsrtr/HPE-Accordion-Component?node-id=71%3A25"
         >
           <AccordionExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          When seeking to provide maximum content in limited, vertical space, an
+          accordion is a good alternative.
+        </SubsectionText>
+        <SubsectionText>
+          An accordion is a group of vertically stacked collapsible buttons with
+          title headers. This gives the user a high level idea of what content
+          will be disclosed before collapsing a section.
+        </SubsectionText>
+        <SubsectionText>
+          Accordions make it easy for the user to process and find the
+          information they need in a more effective manner. Accordions can be
+          used to organize information, which helps the user consume the
+          information they need at a faster pace.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="About Accordion" level={3} gap="small">
+        <SubsectionText>
+          Accordion has two states:
+          <BulletedList level={3} items={['Collapsed', 'Expanded']} />
+          The chevron is used to identify the expand or collapse action while
+          the entire header can be clicked to expand or collapse content. The
+          default behavior is for all of the panels to be closed, with the user
+          only having visual to the heading.
+        </SubsectionText>
+        <SubsectionText>
+          The accordion labels should be kept short and to the point of the
+          information following in each panel. Lengthy panel content will want
+          to be avoided.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          Buttons are using for the header which make the panel header tabable
+          and accessible by screen readers. This makes having a very descriptive
+          heading label helps so the user gets to the content section they are
+          intrested faster. Users are able to click anywhere within the panel
+          heading for the state to change to expanded.
+        </SubsectionText>
+        <SubsectionText>
+          Only one panel can be opened at a time to help with readability.
```

Comment:
I think there is a prop that allows multiple to be opened at once, what if we rephrase to:

To help with readability, accordions should only have one panel open at a time.

Reply:
I think the idea was that they didnt want to tell users of that prop obviously anyone who goes to grommet would see but I can change it